site/app/templates/admin/Configuration.twig

```diff
@@ -178,7 +178,8 @@
             <br>
             <span class="option-alt">
                 Initial number of late days given to all students at the start of the course.
-                Additional late days can be granted (e.g., as incentives) using the "Late Days Allowed" form.
+                Additional late days can be granted (e.g., as incentives) using the "Late Days Allowed" form. <br>
+                <strong> Maximum Late Days : 10,000 </strong> <br>
```

Comment:
This change just makes the UI more cluttered, and is entirely meaningless to most instructors.  The API error response should be more than enough to determine what went wrong if for some reason an instructor decides to award more than 27 years worth of late days...